map.py

```diff
@@ -44,4 +32,24 @@ def generate_map(self):
                     point1.sw = min(point1.sw, distance)
                     point2.ne = min(point2.ne, distance)
 
-        return self.points
\ No newline at end of file
+
+
+    def generate(self):
+        """
+        A function that generates random points with their coordinates and limits
+        with Map objet's parametars
+        
+        """
+        random.seed(self.seed)
+        i = 0
+        while i < self.num_of_points:
```

Comment:
Ovde mozes da proveravas samo `len(self.points) < self.num_of_points`, a posto je `self.points` `set` mozes samo da dodajes `point`-ove u njega, on sam proverava dal su isti.

Takodje nisam siguran kako `set` funkcionise sa custom klasama, mozda ces morati da definises operator `=` za klasu `point` da bi on sve to radio automatski.

Reply:
odradjeno sve